FhirToDataLake/src/Microsoft.Health.Fhir.Synapse.HealthCheck/HealthCheckerRegistrationExtension.cs

```diff
@@ -19,6 +23,24 @@ public static class HealthCheckerRegistrationExtension
 
             services.AddSingleton<IHealthChecker, AzureBlobStorageHealthChecker>();
 
+            SchemaConfiguration schemaConfiguration;
+            try
+            {
+                schemaConfiguration = services
+                    .BuildServiceProvider()
+                    .GetRequiredService<IOptions<SchemaConfiguration>>()
+                    .Value;
+            }
+            catch (Exception ex)
+            {
+                throw new ConfigurationErrorException("Failed to parse schema configuration", ex);
```

Comment:
We parse the configuration before using it? Or it is a standard IOptions config?

Reply:
Feel this already be validated when initialization

Reply:
Configuration has been parsed and validated before it.